source/_components/influxdb.markdown

```diff
@@ -25,28 +25,91 @@ influxdb:
-- **password** (*Optional*): The password for the database user account.
-- **database** (*Optional*): Name of the database to use. Defaults to `home_assistant`. The database must already exist.
-- **ssl** (*Optional*): Use https instead of http to connect. Defaults to false.
-- **verify_ssl** (*Optional*): Verify SSL certificate for https request. Defaults to false.
-- **max_retries** (*Optional*): Allow the component to retry if there was a network error when transmitting data
-- **default_measurement** (*Optional*): Measurement name to use when an entity doesn't have a unit. Defaults to entity id.
-- **override_measurement** (*Optional*): Measurement name to use instead of unit or default measurement. This will store all data points in a single measurement.
-- **component_config**, **component_config_domain**, **component_config_glob** (*Optional*): These attributes contains component-specific override values. See [Customizing devices and services](/getting-started/customizing-devices/) for format.
-  - **override_measurement** (*Optional*): Measurement name to use for this component, takes precedence over the global 'override_measurement' and component-specific 'unit_of_measurement' attribute.
-- **exclude** (*Optional*): Configure which components should be excluded from recording to InfluxDB.
-  - **entities** (*Optional*): The list of entity ids to be excluded from recording to InfluxDB.
-  - **domains** (*Optional*): The list of domains to be excluded from recording to InfluxDB.
-- **include** (*Optional*): Configure which components should be included in recordings to InfluxDB. If set, all other entities will not be recorded to InfluxDB. Values set by the **blacklist** option will prevail.
-  - **entities** (*Optional*): The list of entity ids to be included from recordings to InfluxDB.
-  - **domains** (*Optional*): The list of domains to be included from recordings to InfluxDB.
-- **tags** (*Optional*): Tags to mark the data.
-- **tags_attributes** (*Optional*): The list of attribute names which should be reported as tags and not fields to InfluxDB. For example, if set to `friendly_name`, it will be possible to group by entities' friendly names as well, in addition to their ids.
+{% configuration %}
+host:
+  type: string
+  description: IP address of your database host, e.g., 192.168.1.10
+  required: false
+  default: localhost
+port:
+  type: int
+  description: Port to use
+  required: false
+  default: 8086
+username:
+  type: string
+  description: The username of the database user. The user needs read/write privileges on the database
+  required: false
+password:
+  type: string
+  description: The password for the database user account.
+  required: false
+database:
+  type: string
+  description: Name of the database to use. The database must already exist.
+  required: false
+  default: will use the default database of your influxdb instance
```

Comment:
According to the code (maybe I misread it), the default is `home_assistant`.

https://github.com/home-assistant/home-assistant/blob/e903f7ffdafc7f5e0797f654c9b8314529412775/homeassistant/components/influxdb.py#L40

Reply:
Yes, I second guessed myself and misread the python. Thanks for catching that!